### PR TITLE
Wip: Implement crypto, stocks, and DAF pages

### PR DESCRIPTION
**Pull request by Fishbakh-N**
**Screenshots or Screencaps**

DAF

stocks

crypto

### CODE REVIEW

File: packages/donate-button-v4/src/components/widget/types/Crypto.ts [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+	SAMO = 'SAMO',
+	SBR = 'SBR',
+	SHIB = 'SHIB',
+	SOL = 'SOL',
+	UNI = 'UNI',
+	USDC = 'USDC',
+	USDT = 'USDT',
+	// Mistakenly thought these were supported by TGB
+	// XLM = "XLM",
+	// XMR = "XMR",
```
Comment (rahulgi) [inline packages/donate-button-v4/src/components/widget/types/Crypto.ts:27]:
```suggestion
```

File: packages/donate-button-v4/src/components/widget/types/Crypto.ts [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+	// [CryptoCurrency.XLM]: {
+	//   displayName: "Stellar",
+	//   abbreviation: "XLM",
+	//   decimalOffset: 7,
+	// },
+	// [CryptoCurrency.XMR]: {
+	//   displayName: "Monero",
+	//   abbreviation: "XMR",
+	//   decimalOffset: 12,
+	// },
```
Comment (rahulgi) [inline packages/donate-button-v4/src/components/widget/types/Crypto.ts:195]:
```suggestion
```

File: packages/donate-button-v4/src/resetCss.ts [hunk 3]
```diff
@@ -17,6 +18,7 @@ const resetcss = `
     border: 0;
     font-size: 100%;
     font: inherit;
+    font-family: 'Basis Grotesque Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', Roboto, sans-serif;
```
Comment (rahulgi) [inline packages/donate-button-v4/src/resetCss.ts:21]: Do we actually load our font for the donate-button?

File: packages/donate-button-v4/src/components/widget/types/Crypto.ts [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+export interface SharedCryptoCurrencyConfig {
+	displayName: string;
+	abbreviation: string;
+	decimalOffset: number;
+	contractType?: ContractType;
+}
+
+export const DISABLED_TOKENS = [
+	// Wyre may or may not be dying - check back in in a few months
+	// (comment by rahul, 2023-01-24)
```
Comment (rahulgi) [inline packages/donate-button-v4/src/components/widget/types/Crypto.ts:46]:
```suggestion
```

**Review by rahulgi — APPROVED** [review]

File: packages/donate-button-v4/src/resetCss.ts (hunk 3 above)
Comment (Fishbakh-N) [reply packages/donate-button-v4/src/resetCss.ts:21]: https://github.com/everydotorg/donate-button/blob/7e0c81bebee523493f8b872dbb9cdc676a51b849/packages/donate-button-v4/src/index.tsx#L24
https://github.com/everydotorg/donate-button/blob/7e0c81bebee523493f8b872dbb9cdc676a51b849/packages/donate-button-v4/src/loadFonts.ts#L1-L30

**Pull request merged**